src/components/__tests_/HomeCard.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import HomeCard from '../HomeCard';
+
+it('Test to see if the home card renders well ', () => {
+  render(<HomeCard imageUrl="url" country="Uganda" aqi={3} />);
+  const renderedText = screen.getByText(/Air Quality Index/i);
+  expect(renderedText).toBeInTheDocument();
+});
```

Comment:
- [ ] The HomeCard component rendering test is failing in this file. Kindly work on it please 👍 

![image](https://user-images.githubusercontent.com/50941074/194719983-77603036-a989-43ea-bbbd-ce280db14d19.png)